index.html

```diff
@@ -280,28 +222,63 @@ <h4 class="frame-name">CSS</h4>
             <li class="tabs">
               <h4 class="tab-title">Framework</h4>
               <a href="#">
-
                 <img src="./image/right.svg" alt="right" class="right-arrow" />
-
-                <img src="./image/right.png" alt="right" class="right-arrow" />
-
               </a>
             </li>
           </ul>
           <ul class="pro-languages separator-bottom">
             <li class="tabs">
               <h4 class="tab-title">Skills</h4>
               <a href="#">
-
                 <img src="./image/right.svg" alt="right" class="right-arrow" />
-
-                <img src="./image/right.png" alt="right" class="right-arrow" />
-
               </a>
             </li>
           </ul>
         </div>
       </div>
     </section>
+    <section class="form">
+      <ul class="form-header">
+        <li class="form-list">
+          <h4 class="contact-title">Contact me</h4>
+          <p class="contact-description">
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I’d love
+            to help with it
+          </p>
+        </li>
+      </ul>
+      <form action="https://formspree.io/f/mgebvkpz" method="POST"  class="form-details">
+        <input
+        name="name"
+          class="username"
+          type="text"
+          id="name"
+          placeholder="Yeremias"
+          max-length="30"
+          required
+        />
+        <input 
+        name="email"
+         class="useremail"
+          type="email"
+          id="email"
+          placeholder="natajayanj14@gmail.com"
+          required
+        />
+        <textarea
+        class="userfeedback"
+          name="feedback"
+          id="feedback"
+          cols="30"
+          rows="10"
+          max-length="500"
+          placeholder="Write your message here"
+          required
+        ></textarea>
```

Comment:
- [ ] The same applies here, it should be `maxlength`. Please correct the typo! 🤓 🔧 

```suggestion
        <textarea
        class="userfeedback"
          name="feedback"
          id="feedback"
          cols="30"
          rows="10"
          maxlength="500"
          placeholder="Write your message here"
          required
        ></textarea>
```